dist/stat_data.py

```diff
@@ -397,6 +397,10 @@ def __init__(self, name, desc, flags=''):
     CheckpointStat('checkpoint_handle_duration', 'most recent duration for gathering all handles (usecs)', 'no_clear,no_scale'),
     CheckpointStat('checkpoint_handle_duration_apply', 'most recent duration for gathering applied handles (usecs)', 'no_clear,no_scale'),
     CheckpointStat('checkpoint_handle_duration_skip', 'most recent duration for gathering skipped handles (usecs)', 'no_clear,no_scale'),
+    CheckpointStat('checkpoint_handle_meta_check', 'most recent handles metadata checked'),
+    CheckpointStat('checkpoint_handle_meta_check_duration', 'most recent duration for handles metadata checked (usecs)', 'no_clear,no_scale'),
+    CheckpointStat('checkpoint_handle_lock', 'most recent handles metadata locked'),
+    CheckpointStat('checkpoint_handle_lock_duration', 'most recent duration for locking the handles (usecs)', 'no_clear,no_scale'),
```

Comment:
I think the time most likely is spent in `__wt_checkpoint_get_handles`. Can we add some stats there?